php-nginx/entrypoint.sh

```diff
@@ -138,4 +134,7 @@ chmod -R 550 ${DOCUMENT_ROOT}
 # Change the www-data's shell back to /usr/sbin/nologin
 chsh -s /usr/sbin/nologin www-data
 
+# Remove the loose php-cli.ini
+rm ${PHP_DIR}/lib/php-cli.ini
```

Comment:
but we're locking down `exec` specifically so this doesn't happen. Until we can actually output a proper error response on the CLI, we really can't keep suhosin activated.